packages/frontend/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "oss-automation"
```

Comment:
we'll want to make this a variable eventually, so that it's easy to deploy to other projects.